src/emailer.js

```diff
@@ -344,9 +344,13 @@ function buildCustomTemplates(config) {
 			}, next);
 		},
 		function (result, next) {
-			var templates = result.templates.filter(function (template) {
-				return template.isCustom && template.text !== prevConfig['email:custom:' + path];
-			});
+			// If the new config contains any email override values, re-compile those templates
+			var toBuild = Object
+				.keys(config)
+				.filter(prop => prop.startsWith('email:custom:'))
+				.map(key => key.split(':')[2]);
+
+			var templates = result.templates.filter(template => toBuild.includes(template.path));
```

Comment:
The logic change here no longer checks the template for equality, and instead simply re-compiles them if they are present in the passed-in `config`. As it turns out, `config` as passed in only contains the values changes, so this is sufficient for this purpose anyway.